### DISCUSSION TITLE
i think the project need to be renamed to be distinguished from “database”

### DISCUSSION THREAD

**Discussion by baby5** [encode/databases#346] [Ideas]
i search "python databases" on google, you know, i do not get what i want. maybe a good name will be better.

aiodbs/asyncdbs maybe a good one

**Comment by Kludex** [comment]
![image](https://user-images.githubusercontent.com/7353520/120294263-4f9e0000-c2c6-11eb-99bc-712db0302e70.png)

**Reply by baby5** [reply]
not only pypi, if i want to get some articles about "databases", how does google distinguish.

**Reply by houmie** [reply]
It's true. The name couldn't be more generic. Try to tag that name on Stackoverflow, it won't be possible.
